premailer/premailer.py

```diff
@@ -161,7 +106,8 @@ def __init__(self, html, base_url=None,
                  method="html",
                  base_path=None,
                  disable_basic_attributes=None,
-                 disable_validation=False):
+                 disable_validation=False,
+                 cache_css_parsing=True):
```

Comment:
Yes, it was a hard decision to put True on that kwarg. I always took other to not break the api when adding new stuffs. But this time I added true since this is mostly silent and the chance that someone could have problem with memory is small. But they will have some improvement.

We can set to False if you like but I think it may worth the risk